Windows.MSVC.toolchain.cmake

```diff
@@ -37,6 +37,7 @@
 # | CMAKE_VS_PLATFORM_TOOLSET_VERSION           | The version of the MSVC toolset to use. For example, 14.29.30133. Defaults to the highest available.                     |
 # | CMAKE_VS_PLATFORM_TOOLSET_HOST_ARCHITECTURE | The architecture of the toolset to use. Defaults to 'x64'.                                                               |
 # | CMAKE_WINDOWS_KITS_10_DIR                   | The location of the root of the Windows Kits 10 directory.                                                               |
+# | VS_ENABLE_SPECTRE                           | Whether to compile/link with 'Spectre' mitigations enable. Defaults to 'ON' for non-Debug CMAKE_BUILD_TYPE builds.       |
```

Comment:
I wonder if `VS_ENABLE_SPECTRE_MITIGATIONS` might be more precise. We're trying to prevent Spectre here 😉.

Reply:
👍